Formula/wandio.rb

```diff
@@ -1,10 +1,9 @@
 class Wandio < Formula
   desc "Transparently read from and write to zip, bzip2, lzma or zstd archives"
   homepage "https://research.wand.net.nz/software/libwandio.php"
-  url "https://research.wand.net.nz/software/wandio/wandio-4.2.3.tar.gz"
-  sha256 "78c781ce2c3783b85d894e29005b7e98fc246b33f94616047de3bb4d11d4d823"
+  url "https://github.com/LibtraceTeam/wandio/archive/refs/tags/4.2.4-1.tar.gz"
```

Comment:
Original homepage (https://web.archive.org/web/20220305181150/https://research.wand.net.nz/software/libwandio.php) references wiki at https://github.com/wanduow/wandio/wiki. This redirects to new org.

Not sure about version number. `4.2.4-1` tag may be to match Debian equivalent of `revision`